classes/submission/maps/Schema.php

```diff
@@ -445,6 +445,7 @@ protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, b
                 'roundId' => (int) $reviewAssignment->getReviewRoundId(),
                 'recommendation' => $reviewAssignment->getRecommendation(),
                 'dateCancelled' => $reviewAssignment->getData('dateCancelled'),
+                'reviewerId' => $anonymizeReviews && $anonymizeReviews->contains($reviewAssignment->getId()) ? null : $reviewAssignment->getId(),
                 'reviewerFullName' => $anonymizeReviews && $anonymizeReviews->contains($reviewAssignment->getId()) ? '' : $reviewAssignment->getData('reviewerFullName'),
```

Comment:
I am using submission.reviewAssignments to list the reviewers. And when adding actions I also needed reviewerId, to be able to "Login as".

Reply:
`'reviewerId' => ... $reviewAssignment->getReviewerId()`?